<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Runs a TINC analysis. — autofit • TINC</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Runs a TINC analysis. — autofit"><meta property="og:description" content="This function is a wrapper to run the main analysis of TINC.
The steps are as follows:
1) Input data is loaded from a `file`, or from a `dataframe`.
2) Clonal mutations are estimated for the tumour, together with
the tumour purity (Tumour in Tumour).
3) From putative clonal mutations of the tumour, the Tumour in Normal
contamination level is estimated.
An S3 object is returned that contains the results of the analysis."><meta property="og:image" content="caravagnalab.github.io/TINC/logo.png"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TINC</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/TINC.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/Analysis.html">TINC analysis</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/caravagnalab/TINC/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Runs a TINC analysis.</h1>
    <small class="dont-index">Source: <a href="https://github.com/caravagnalab/TINCR/autofit.R" class="external-link"><code>R/autofit.R</code></a></small>
    <div class="hidden name"><code>autofit.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This function is a wrapper to run the main analysis of TINC.</p>
<p>The steps are as follows:</p><ul><li><p>1) Input data is loaded from a `file`, or from a `dataframe`.</p></li>
<li><p>2) Clonal mutations are estimated for the tumour, together with
the tumour purity (Tumour in Tumour).</p></li>
<li><p>3) From putative clonal mutations of the tumour, the Tumour in Normal
contamination level is estimated.</p></li>
</ul><p>An S3 object is returned that contains the results of the analysis.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">autofit</span><span class="op">(</span></span>
<span>  <span class="va">input</span>,</span>
<span>  <span class="va">cna</span>,</span>
<span>  VAF_range_tumour <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">0</span>, <span class="fl">0.7</span><span class="op">)</span>,</span>
<span>  cutoff_miscalled_clonal <span class="op">=</span> <span class="fl">0.6</span>,</span>
<span>  cutoff_lv_assignment <span class="op">=</span> <span class="fl">0.75</span>,</span>
<span>  N <span class="op">=</span> <span class="fl">20000</span>,</span>
<span>  FAST <span class="op">=</span> <span class="cn">FALSE</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div id="arguments">
    <h2>Arguments</h2>
    <dl><dt>input</dt>
<dd><p>A`dataframe` of the iput mutations. Must be in a certain format,
see the vignette for more information.</p></dd>


<dt>cna</dt>
<dd><p>Copy Number data in the format of package <code>CNAqc</code>.</p></dd>


<dt>VAF_range_tumour</dt>
<dd><p>A range `[x, y]` so that only mutations
with VAF in that range are actually used to determine the TIN/ TIT
levels of the input.</p></dd>


<dt>cutoff_miscalled_clonal</dt>
<dd><p>An upper bound on the VAF of a
cluster in the tumour data. Clusters above this value will be
considered miscalled clonal clusters (e.g., due to LOH etc.).</p></dd>


<dt>cutoff_lv_assignment</dt>
<dd><p>Consider only latent variables with
responsibilities above this cutoff.</p></dd>


<dt>N</dt>
<dd><p>If there are more than `N` mutations in VAF range
`VAF_range_tumour`, a random subset of size `N` is retained.</p></dd>


<dt>FAST</dt>
<dd><p>If `TRUE`, it runs the analysis with reduced sampling
power and accuracy. Use this to obtain a result for preliminary
inspection of your data, and then run `autofit` with this parameter
set to `FALSE`.</p></dd>

</dl></div>
    <div id="value">
    <h2>Value</h2>
    

<p>An S3 object that contains the results of this analysis.</p>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># Random</span></span></span>
<span class="r-in"><span><span class="va">rt</span> <span class="op">=</span> <span class="fu"><a href="random_TIN.html">random_TIN</a></span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> Generated TINC dataset (n = 991 mutations), TIN (0.05) and TIT (1), normal and tumour coverage 30x and 120x.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Removed 2 rows containing missing values (`geom_bar()`).</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Removed 2 rows containing missing values (`geom_bar()`).</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="va">x</span> <span class="op">=</span> <span class="fu">autofit</span><span class="op">(</span>input <span class="op">=</span> <span class="va">rt</span><span class="op">$</span><span class="va">data</span>, cna <span class="op">=</span> <span class="va">rt</span><span class="op">$</span><span class="va">cna</span>, FAST <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #BBBBBB; background-color: #0000BB;"> [ TINC ] </span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Loading TINC input data</span> <span style="color: #00BBBB;">─────────────────────────────────────────────────────</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> Input data contains n = 991 mutations, selecting operation mode.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #BBBB00;">!</span> Found CNA data, retaining only mutations that map to segments with predominant karyotype ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">CNAqc - CNA Quality Check</span> <span style="color: #00BBBB;">───────────────────────────────────────────────────</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Using reference genome coordinates for: <span style="color: #00BB00;">GRCh38</span>.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> Fortified calls for <span style="color: #00BB00;">991</span> somatic mutations: <span style="color: #00BB00;">991</span> SNVs (<span style="color: #00BB00;">100%</span>) and <span style="color: #00BB00;">0</span> indels.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #BBBB00;">!</span> CNAs have no CCF, assuming clonal CNAs (CCF = 1).</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #BBBB00;">!</span> Added segments length (in basepairs) to CNA segments.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> Fortified CNAs for <span style="color: #00BB00;">991</span> segments: <span style="color: #00BB00;">991</span> clonal and <span style="color: #00BB00;">0</span> subclonal.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>[CNAqc] a karyotype column is present in CNA calls, and will be overwritten</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="color: #00BB00;">991</span> mutations mapped to clonal CNAs.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ── <span style="font-weight: bold;">Genome coverage by karyotype, in basepairs.</span> ──</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #949494;"># A tibble: 1 × 4</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   minor Major     n karyotype</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>    </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #BCBCBC;">1</span>     1     1  <span style="text-decoration: underline;">2</span>973 1:1      </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> n = 991 mutations mapped to CNA segments with karyotype <span style="color: #00BB00;">1:1</span> (largest available in basepairs).</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> Mutation with VAF within 0 and 0.7 ~ n = 986.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Analysing tumour sample with MOBSTER</span> <span style="color: #00BBBB;">────────────────────────────────────────</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #BBBBBB; background-color: #0000BB;"> [ MOBSTER fit ] </span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> Loaded input data, n = 986.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ❯ n = 986. Mixture with k = <span style="color: #00BB00;">1,2</span> Beta(s). Pareto tail: <span style="color: #00BB00;">TRUE</span> and <span style="color: #00BB00;">FALSE</span>. Output</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> clusters with π &gt; 0.02 and n &gt; 10.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #BBBB00;">!</span> mobster automatic setup <span style="color: #00BB00;">FAST</span> for the analysis.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ❯ Scoring (<span style="color: #BB0000;">without parallel</span>) 2 x 2 x 2 = <span style="color: #00BB00;">8</span> models by <span style="color: #00BB00;">reICL</span>.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #BBBBBB;">ℹ</span> <span style="font-weight: bold;">MOBSTER fits</span> completed in 6.8s.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ── <span style="color: #000000; background-color: #BBBB00;">[ MOBSTER ] My MOBSTER model</span> n = 986 with <span style="color: #00BB00;">k = 1</span> Beta(s) <span style="color: #00BB00;">and a tail</span> ──────────</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ● Clusters: π = 68% [<span style="color: #BBBB00;">C1</span>] and 32% [<span style="color: #BBBB00;">Tail</span>], with π &gt; 0.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ● Tail [n = 305, 32%] with alpha = 1.1.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ● Beta <span style="color: #00BB00;">C1</span> [n = 681, 68%] with mean = 0.49.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #BBBBBB;">ℹ</span> Score(s): NLL = -960.87; ICL = -1814.51 (-1880.37), H = 65.86 (0). Fit</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">converged</span> by <span style="color: #00BB00;">MM</span> in 13 steps.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> With CNA, TINC will estimating tumour purity adjusting by copy number and mutation multiplicity.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Mutant allele copies <span style="color: #00BB00;">1</span> for karyotype <span style="color: #00BB00;">1:1</span></span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>You did not pass enough input colours, adding a gray colour</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> Available: C1, Tail</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> Missing: NA</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>The `&lt;scale&gt;` argument of `guides()` cannot be `FALSE`. Use "none" instead as</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> of ggplot2 3.3.4.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> The deprecated feature was likely used in the <span style="color: #0000BB;">mobster</span> package.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span>   Please report the issue at <span style="color: #0000BB; font-style: italic;">&lt;https://github.com/caravagnalab/mobster/issues&gt;</span>.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>The dot-dot notation (`..count..`) was deprecated in ggplot2 3.4.0.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Please use `after_stat(count)` instead.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> The deprecated feature was likely used in the <span style="color: #0000BB;">mobster</span> package.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span>   Please report the issue at <span style="color: #0000BB; font-style: italic;">&lt;https://github.com/caravagnalab/mobster/issues&gt;</span>.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> MOBSTER found n = 674 clonal mutations from cluster C1</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Analysing normal sample with BMix</span> <span style="color: #00BBBB;">───────────────────────────────────────────</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">BMix fit</span> <span style="color: #00BBBB;">────────────────────────────────────────────────────────────────────</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Binomials k_B = <span style="color: #00BB00;">1</span> and <span style="color: #00BB00;">2</span>, Beta-Binomials k_BB = <span style="color: #00BB00;">0</span>; 4 fits to run.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> <span style="font-weight: bold;">Bmix best fit</span> completed in 0 mins</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ── <span style="color: #000000; background-color: #BBBB00;">[ BMix ] My BMix model</span> n = 674 with <span style="color: #00BB00;">k = 1</span> component(s) (<span style="color: #00BB00;">1</span> + <span style="color: #00BB00;">0</span>) ──────────────</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> • Clusters: π = 100% [<span style="color: #BBBB00;">Bin 1</span>], with π &gt; 0.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> • Binomial <span style="color: #00BB00;">Bin 1</span> with mean = 0.00188118811881188.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Score (model selection): <span style="color: #00BB00;">ICL</span> = 756.42.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Scale for <span style="color: #00BB00;">x</span> is already present.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Adding another scale for <span style="color: #00BB00;">x</span>, which will replace the existing scale.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Scale for <span style="color: #00BB00;">fill</span> is already present.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Adding another scale for <span style="color: #00BB00;">fill</span>, which will replace the existing scale.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Removed 2 rows containing missing values (`geom_bar()`).</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Removed 551 rows containing missing values (`geom_raster()`).</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> Binomial peaks 0.00188118811881188 with proportions 1. Clonal score 0.00188118811881188 with TINN 0.00376237623762376</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Analysing tumour and normal samples with VIBER</span> <span style="color: #00BBBB;">──────────────────────────────</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #BBBBBB; background-color: #0000BB;"> [ VIBER - variational fit ] </span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Input n = 986, with k &lt; 5. Dirichlet concentration <span style="color: #00BB00;">α = 1e-06</span>.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Beta (a_0, b_0) = (1, 1); q_i = prior. Optimise: ε = 1e-06 or 1000 steps, r = 3 starts.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> [easypar] 2023-06-22 17:11:32.822746 - Overriding parallel execution setup [TRUE] with global option : FALSE</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> <span style="font-weight: bold;">VIBER fit</span> completed in 0.02 mins (status: <span style="color: #00BB00;">converged</span>)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ── <span style="color: #000000; background-color: #BBBB00;">[ VIBER ] My VIBER model</span> n = 986 (w = 2 dimensions). Fit with <span style="color: #00BB00;">k = 5</span> clusters.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> • Clusters: π = 70% [<span style="color: #BBBB00;">C4</span>], 23% [<span style="color: #BBBB00;">C1</span>], and 7% [<span style="color: #BBBB00;">C5</span>], with π &gt; 0.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> • Binomials: θ = &lt;0, 0.49&gt; [<span style="color: #BBBB00;">C4</span>], &lt;0, 0.08&gt; [<span style="color: #BBBB00;">C1</span>], and &lt;0, 0.2&gt; [<span style="color: #BBBB00;">C5</span>].</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Score(s): ELBO = -137383.715. Fit <span style="color: #00BB00;">converged</span> in 30 steps, ε = 1e-06.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BB00;">✔</span> Reduced to k = 3 (from 5) selecting VIBER cluster(s) with π &gt; 0.02, and Binomial p &gt; 0 in w &gt; 0 dimension(s).</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>The `x` argument of `as_tibble.matrix()` must have unique column names if</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> `.name_repair` is omitted as of tibble 2.0.0.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Using compatibility `.name_repair`.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> The deprecated feature was likely used in the <span style="color: #0000BB;">VIBER</span> package.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span>   Please report the issue at <span style="color: #0000BB; font-style: italic;">&lt;https://github.com/caravagnalab/VIBER/issues&gt;</span>.</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">x</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">TINC profiler for bulk WGS</span> <span style="color: #00BBBB;">──────────────────────────────────────────────────</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Copy Number data has been used for this analysis (karyotype <span style="color: #00BB00;">1:1</span>)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ────────────────────────────────────────────────────────────────────────────────</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ── <span style="color: #000000; background-color: #BBBB00;">[ CNAqc ] MySample</span> <span style="color: #00BB00;">991</span> mutations in <span style="color: #00BB00;">991</span> segments (<span style="color: #00BB00;">991</span> clonal, <span style="color: #00BB00;">0</span> subclonal). G</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ── Clonal CNAs </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  1:1  [n = 991, L =   0 Mb] ■■■■■■■■■■■■■■■■■■■■■■■■■■■</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Sample Purity: 80% ~ Ploidy: 2.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ────────────────────────────────────────────────────────────────────────────────</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> → Mutations data: n = <span style="color: #00BB00;">986</span> out of 991 within range (<span style="color: #00BB00;">99%</span>).</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #000000; background-color: #BBBB00;">        TIT : </span> 97% (RF 97)  ~ n = 674 clonal mutations, cluster C1 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #000000; background-color: #BBBB00;">        TIN : </span> 0% (RF 0)  ~ n = 38 with VAF &gt; 0 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #000000; background-color: #00BB00; font-weight: bold;">   QC Tumour  </span> <span style="color: #00BB00;">High purity (&gt;85%)</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> <span style="color: #000000; background-color: #00BB00; font-weight: bold;">   QC Normal  </span> <span style="color: #00BB00;">No Contamination (&lt;1%)</span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># Fit in the package</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="st">'fit_example'</span>, package <span class="op">=</span> <span class="st">'TINC'</span><span class="op">)</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">fit_example</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">TINC profiler for bulk WGS</span> <span style="color: #00BBBB;">──────────────────────────────────────────────────</span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> <span style="color: #00BBBB;">ℹ</span> Copy Number data has been used for this analysis (karyotype <span style="color: #00BB00;">1:1</span>)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ────────────────────────────────────────────────────────────────────────────────</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ── <span style="color: #000000; background-color: #BBBB00;">[ CNAqc ] </span>  mutations in <span style="color: #00BB00;">985</span> segments (<span style="color: #00BB00;">985</span> clonal,  subclonal). Genome refere</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> </span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> ── Clonal CNAs </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  1:1  [n = 985, L =   0 Mb] ■■■■■■■■■■■■■■■■■■■■■■■■■■■</span>
<span class="r-err co"><span class="r-pr">#&gt;</span> <span class="error">Error in UseMethod("filter"):</span> no applicable method for 'filter' applied to an object of class "NULL"</span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by <a href="https://www.caravagnalab.org" class="external-link">Giulio Caravagna</a>.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer></div>

  


  

  </body></html>

