<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>TINC analysis • TINC</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="TINC analysis">
<meta property="og:description" content="TINC">
<meta property="og:image" content="caravagnalab.github.io/TINC/logo.png">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TINC</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/TINC.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/Analysis.html">TINC analysis</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/caravagnalab/TINC/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="Analysis_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>TINC analysis</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/caravagnalab/TINCvignettes/Analysis.Rmd"><code>vignettes/Analysis.Rmd</code></a></small>
      <div class="hidden name"><code>Analysis.Rmd</code></div>

    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/caravagn/TINC">TINC</a></span><span class="op">)</span></code></pre></div>
<div id="analysis-pipeline" class="section level1">
<h1 class="hasAnchor">
<a href="#analysis-pipeline" class="anchor"></a>Analysis pipeline</h1>
<p>The TINC pipelines requires to run a single function: <code>autofit</code>.</p>
<p>Results can then be inspected using two report functions (see below).</p>
</div>
<div id="example-simulated-inputs" class="section level1">
<h1 class="hasAnchor">
<a href="#example-simulated-inputs" class="anchor"></a>Example simulated inputs</h1>
<p>TIN provides a function to generate a random dataset to test the tool. The input <code>TIN</code> and <code>TIT</code> parameters can be specified.</p>
<blockquote>
<p>Note: the sampler generates VAFs using <a href="https://caravagnalab.github.io/mobster/">MOBSTER</a>, and then samples some clonal mutations and contaminates the normal. This process is noisy and might return slightly lower values for the actual <code>TIT</code> estimate. The best setup to test TINC is with a simulated BAM file - e.g., created via <a href="https://github.com/adamewing/bamsurgeon">BAMSurgeon</a>.</p>
</blockquote>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/Random.html">set.seed</a></span><span class="op">(</span><span class="fl">1234</span><span class="op">)</span>

<span class="co"># Sample data from a tumour with 80% purity, and 10% contamination in the normal</span>
<span class="va">sampled_data</span> <span class="op">=</span> <span class="fu"><a href="../reference/random_TIN.html">random_TIN</a></span><span class="op">(</span>TIT <span class="op">=</span> <span class="fl">.8</span>, TIN <span class="op">=</span> <span class="fl">0.1</span><span class="op">)</span>

<span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">sampled_data</span><span class="op">)</span>
<span class="co">#&gt; $data</span>
<span class="co">#&gt; # A tibble: 995 × 14</span>
<span class="co">#&gt;    chr        from       to ref   alt   simulated_clust… n_tot_count t_tot_count</span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;int&gt;    &lt;dbl&gt; &lt;chr&gt; &lt;chr&gt; &lt;chr&gt;                  &lt;int&gt;       &lt;int&gt;</span>
<span class="co">#&gt;  1 chr2   79428815   7.94e7 C     T     C1                        24         138</span>
<span class="co">#&gt;  2 chr5  115979395   1.16e8 C     A     C1                        23         120</span>
<span class="co">#&gt;  3 chr4   19850350   1.99e7 C     C     C1                        33         132</span>
<span class="co">#&gt;  4 chr16  79585444   7.96e7 C     C     C1                        37         114</span>
<span class="co">#&gt;  5 chr10  75573241   7.56e7 A     G     C1                        33         131</span>
<span class="co">#&gt;  6 chr20  56028967   5.60e7 G     C     C1                        29         118</span>
<span class="co">#&gt;  7 chr7   47030220   4.70e7 A     G     C1                        29         103</span>
<span class="co">#&gt;  8 chr21  20153237   2.02e7 T     T     C1                        38         135</span>
<span class="co">#&gt;  9 chr7  116719847   1.17e8 T     A     C1                        33         109</span>
<span class="co">#&gt; 10 chr1   56648935   5.66e7 T     T     C1                        31         119</span>
<span class="co">#&gt; # … with 985 more rows, and 6 more variables: n_alt_count &lt;dbl&gt;,</span>
<span class="co">#&gt; #   t_alt_count &lt;dbl&gt;, n_ref_count &lt;dbl&gt;, t_ref_count &lt;dbl&gt;, sim_t_vaf &lt;dbl&gt;,</span>
<span class="co">#&gt; #   sim_n_vaf &lt;dbl&gt;</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $cna</span>
<span class="co">#&gt; # A tibble: 995 × 6</span>
<span class="co">#&gt;    chr        from        to ref   Major minor</span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;dbl&gt;     &lt;dbl&gt; &lt;chr&gt; &lt;dbl&gt; &lt;dbl&gt;</span>
<span class="co">#&gt;  1 chr2   79428814  79428817 C         1     1</span>
<span class="co">#&gt;  2 chr5  115979394 115979397 C         1     1</span>
<span class="co">#&gt;  3 chr4   19850349  19850352 C         1     1</span>
<span class="co">#&gt;  4 chr16  79585443  79585446 C         1     1</span>
<span class="co">#&gt;  5 chr10  75573240  75573243 A         1     1</span>
<span class="co">#&gt;  6 chr20  56028966  56028969 G         1     1</span>
<span class="co">#&gt;  7 chr7   47030219  47030222 A         1     1</span>
<span class="co">#&gt;  8 chr21  20153236  20153239 T         1     1</span>
<span class="co">#&gt;  9 chr7  116719846 116719849 T         1     1</span>
<span class="co">#&gt; 10 chr1   56648934  56648937 T         1     1</span>
<span class="co">#&gt; # … with 985 more rows</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $plot</span></code></pre></div>
<p><img src="Analysis_files/figure-html/unnamed-chunk-2-1.png" width="576"></p>
<p>The sampler generates random SNVs, and plain diploid CNA segments. These segments are not real and span just through each one of the simulated SNVs.</p>
<p>Data mappings - done internally by TINC - are also available via the <code>load_TINC_input</code> function.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/load_TINC_input.html">load_TINC_input</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">sampled_data</span><span class="op">$</span><span class="va">data</span>, cna <span class="op">=</span> <span class="va">sampled_data</span><span class="op">$</span><span class="va">cna</span><span class="op">)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── Loading TINC input data ─────────────────────────────────────────────────────</span>
<span class="co">#&gt; ✔ Input data contains n = 995 mutations, selecting operation mode.</span>
<span class="co">#&gt; ! Found CNA data, retaining only mutations that map to segments with predominant karyotype ...</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;  [ CNAqc - CNA Quality Check ]</span>
<span class="co">#&gt; ℹ Using reference genome coordinates for: GRCh38.</span>
<span class="co">#&gt; ! Missing CCF column from CNA calls, adding CCF = 1 assuming clonal CNA calls.</span>
<span class="co">#&gt; ! Missing segments length from CNA calls, adding it to CNA calls.</span>
<span class="co">#&gt; ℹ Input n = 995 mutations for 995 CNA segments (995 clonal, 0 subclonal)</span>
<span class="co">#&gt; Warning in map_mutations_to_segments(snvs, cna %&gt;% filter(CCF == 1)): [CNAqc] a</span>
<span class="co">#&gt; karyotype column is present in CNA calls, and will be overwritten</span>
<span class="co">#&gt; ✔ Mapped n = 995 mutations to clonal segments (100% of input)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── Genome coverage by karyotype, in basepairs. ──</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; # A tibble: 1 × 4</span>
<span class="co">#&gt;   minor Major     n karyotype</span>
<span class="co">#&gt;   &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt; &lt;chr&gt;    </span>
<span class="co">#&gt; 1     1     1  2985 1:1</span>
<span class="co">#&gt; ✔ n = 995 mutations mapped to CNA segments with karyotype 1:1 (largest available in basepairs).</span>
<span class="co">#&gt; ✔ Mutation with VAF within 0 and 0.7 ~ n = 995.</span>
<span class="co">#&gt; $mutations</span>
<span class="co">#&gt; # A tibble: 995 × 12</span>
<span class="co">#&gt;    chr        from        to ref   alt   n_ref_count n_alt_count t_ref_count</span>
<span class="co">#&gt;    &lt;chr&gt;     &lt;int&gt;     &lt;dbl&gt; &lt;chr&gt; &lt;chr&gt;       &lt;dbl&gt;       &lt;dbl&gt;       &lt;dbl&gt;</span>
<span class="co">#&gt;  1 chr2   79428815  79428816 C     T              24           0          96</span>
<span class="co">#&gt;  2 chr5  115979395 115979396 C     A              22           1          72</span>
<span class="co">#&gt;  3 chr4   19850350  19850351 C     C              32           1          77</span>
<span class="co">#&gt;  4 chr16  79585444  79585445 C     C              36           1          70</span>
<span class="co">#&gt;  5 chr10  75573241  75573242 A     G              32           1          75</span>
<span class="co">#&gt;  6 chr20  56028967  56028968 G     C              28           1          75</span>
<span class="co">#&gt;  7 chr7   47030220  47030221 A     G              28           1          67</span>
<span class="co">#&gt;  8 chr21  20153237  20153238 T     T              37           1          95</span>
<span class="co">#&gt;  9 chr7  116719847 116719848 T     A              32           1          71</span>
<span class="co">#&gt; 10 chr1   56648935  56648936 T     T              30           1          76</span>
<span class="co">#&gt; # … with 985 more rows, and 4 more variables: t_alt_count &lt;dbl&gt;,</span>
<span class="co">#&gt; #   karyotype &lt;chr&gt;, id &lt;chr&gt;, OK_tumour &lt;lgl&gt;</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $cna_map</span>
<span class="co">#&gt; ── [ CNAqc ]  n = 995 mutations in 995 segments (995 clonal + 0 subclonal). Geno</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;  1:1  [n = 995, L =   0 Mb] ■■■■■■■■■■■■■■■■■■■■■■■■■■■</span>
<span class="co">#&gt; ℹ Sample Purity: 80% ~ Ploidy: 2.</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $what_we_used</span>
<span class="co">#&gt; [1] "1:1"</span></code></pre></div>
<p>TINC fits the data via the <code>autofit</code> function.</p>
<p>A flag <code>FAST</code> sets faster set of parameters to determine TIN and TIT scores - we usually this for prototyping analyses and then turn it to <code>FALSE</code>.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Autofit function</span>
<span class="va">TINC_fit</span> <span class="op">=</span> <span class="fu"><a href="../reference/autofit.html">autofit</a></span><span class="op">(</span><span class="va">sampled_data</span><span class="op">$</span><span class="va">data</span>, cna <span class="op">=</span> <span class="cn">NULL</span>, FAST <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span>
<span class="co">#&gt;  [ TINC ]</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── Loading TINC input data ─────────────────────────────────────────────────────</span>
<span class="co">#&gt; ✔ Input data contains n = 995 mutations, selecting operation mode.</span>
<span class="co">#&gt; ✔ Mutation with VAF within 0 and 0.7 ~ n = 995.</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── Analysing tumour sample with MOBSTER ────────────────────────────────────────</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;  [ MOBSTER fit ]</span>
<span class="co">#&gt; ✔ Loaded input data, n = 995.</span>
<span class="co">#&gt; ❯ n = 995. Mixture with k = 1,2 Beta(s). Pareto tail: TRUE and FALSE. Output</span>
<span class="co">#&gt; clusters with π &gt; 0.02 and n &gt; 10.</span>
<span class="co">#&gt; ! mobster automatic setup FAST for the analysis.</span>
<span class="co">#&gt; ❯ Scoring (without parallel) 2 x 2 x 2 = 8 models by reICL.</span>
<span class="co">#&gt; ℹ MOBSTER fits completed in 14.6s.</span>
<span class="co">#&gt; ── [ MOBSTER ] My MOBSTER model n = 995 with k = 1 Beta(s) and a tail ──────────</span>
<span class="co">#&gt; ● Clusters: π = 74% [C1] and 26% [Tail], with π &gt; 0.</span>
<span class="co">#&gt; ● Tail [n = 241, 26%] with alpha = 1.2.</span>
<span class="co">#&gt; ● Beta C1 [n = 754, 74%] with mean = 0.39.</span>
<span class="co">#&gt; ℹ Score(s): NLL = -1093.22; ICL = -2070.49 (-2145.02), H = 74.53 (0). Fit</span>
<span class="co">#&gt; converged by MM in 12 steps.</span>
<span class="co">#&gt; ℹ Using the location likelihood heuristic to inspect mutations' distribution</span>
<span class="co">#&gt; ✔ Cluster C1: 60% of counts in 12 chromosomes (spread over &gt;20% of used chromosomes: 4.4).</span>
<span class="co">#&gt; ℹ Without CNA, TINC will estimate tumour purity as 2*x, with x the clonal peak.</span>
<span class="co">#&gt; ✔ MOBSTER found n = 746 clonal mutations from cluster C1</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── Analysing normal sample with BMix ───────────────────────────────────────────</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── BMix fit ────────────────────────────────────────────────────────────────────</span>
<span class="co">#&gt; ℹ Binomials k_B = 1 and 2, Beta-Binomials k_BB = 0; 4 fits to run.</span>
<span class="co">#&gt; ℹ Bmix best fit completed in 0.01 mins</span>
<span class="co">#&gt; ── [ BMix ] My BMix model n = 746 with k = 1 component(s) (1 + 0) ──────────────</span>
<span class="co">#&gt; • Clusters: π = 100% [Bin 1], with π &gt; 0.</span>
<span class="co">#&gt; • Binomial Bin 1 with mean = 0.0335416572995819.</span>
<span class="co">#&gt; ℹ Score (model selection): ICL = 1700.83.</span>
<span class="co">#&gt; Scale for 'x' is already present. Adding another scale for 'x', which will</span>
<span class="co">#&gt; replace the existing scale.</span>
<span class="co">#&gt; Scale for 'fill' is already present. Adding another scale for 'fill', which</span>
<span class="co">#&gt; will replace the existing scale.</span>
<span class="co">#&gt; ✔ Binomial peaks 0.0335416572995819 with proportions 1. Clonal score 0.0335416572995819 with TINN 0.0670833145991637</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── Analysing tumour and normal samples with VIBER ──────────────────────────────</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;  [ VIBER - variational fit ]</span>
<span class="co">#&gt; ℹ Input n = 995, with k &lt; 5. Dirichlet concentration α = 1e-06.</span>
<span class="co">#&gt; ℹ Beta (a_0, b_0) = (1, 1); q_i = prior. Optimise: ε = 1e-06 or 1000 steps, r = 3 starts.</span>
<span class="co">#&gt; [easypar] 2021-11-22 13:52:25 - Overriding parallel execution setup [TRUE] with global option : FALSE</span>
<span class="co">#&gt; ✔ VIBER fit completed in 0.03 mins (status: converged)</span>
<span class="co">#&gt; ── [ VIBER ] My VIBER model n = 995 (w = 2 dimensions). Fit with k = 5 clusters.</span>
<span class="co">#&gt; • Clusters: π = 78% [C2] and 22% [C1], with π &gt; 0.</span>
<span class="co">#&gt; • Binomials: θ = &lt;0.03, 0.39&gt; [C2] and &lt;0, 0.09&gt; [C1].</span>
<span class="co">#&gt; ℹ Score(s): ELBO = -143389.432. Fit converged in 18 steps, ε = 1e-06.</span>
<span class="co">#&gt; ✔ Reduced to k = 2 (from 5) selecting VIBER cluster(s) with π &gt; 0.02, and Binomial p &gt; 0 in w &gt; 0 dimension(s).</span></code></pre></div>
<p>Summary of the results (with an S3 object)</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/print.html">print</a></span><span class="op">(</span><span class="va">TINC_fit</span><span class="op">)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── TINC profiler for bulk WGS ──────────────────────────────────────────────────</span>
<span class="co">#&gt; ! Copy Number data has not been used for this analysis.</span>
<span class="co">#&gt; → Mutations data: n = 995 out of 995 within range (100%).</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;         TIT :  78% (RF 78)  ~ n = 746 clonal mutations, cluster C1 </span>
<span class="co">#&gt;         TIN :  7% (RF 7)  ~ n = 697 with VAF &gt; 0 </span>
<span class="co">#&gt; </span>
<span class="co">#&gt;    QC Tumour   Good purity (65-85%)</span>
<span class="co">#&gt;    QC Normal   Some contamination (3-7%)</span></code></pre></div>
<p>Plot the results (with an S3 object).</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html">plot</a></span><span class="op">(</span><span class="va">TINC_fit</span><span class="op">)</span></code></pre></div>
<p><img src="Analysis_files/figure-html/unnamed-chunk-6-1.png" width="960"></p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R">
<span class="co"># Analogous to</span>
<span class="co"># plot_simple_report(TINC_fit)</span></code></pre></div>
<p>Or make a more detailed report</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/plot_full_page_report.html">plot_full_page_report</a></span><span class="op">(</span><span class="va">TINC_fit</span><span class="op">)</span></code></pre></div>
<p><img src="Analysis_files/figure-html/unnamed-chunk-7-1.png" width="960"></p>
<p>A similar fit object is available inside the package (called <code>fit_example</code>).</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://www.caravagnalab.org">Giulio Caravagna</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
