<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>TINC analysis • TINC</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="TINC analysis">
<meta property="og:description" content="TINC">
<meta property="og:image" content="caravagnalab.github.io/TINC/logo.png">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TINC</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/TINC.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/Analysis.html">TINC analysis</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/caravagnalab/TINC/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="Analysis_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>TINC analysis</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/caravagnalab/TINCvignettes/Analysis.Rmd" class="external-link"><code>vignettes/Analysis.Rmd</code></a></small>
      <div class="hidden name"><code>Analysis.Rmd</code></div>

    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/caravagn/TINC" class="external-link">TINC</a></span><span class="op">)</span></code></pre></div>
<div class="section level1">
<h1 id="analysis-pipeline">Analysis pipeline<a class="anchor" aria-label="anchor" href="#analysis-pipeline"></a>
</h1>
<p>The TINC pipelines requires to run a single function: <code>autofit</code>.</p>
<p>Results can then be inspected using two report functions (see below).</p>
</div>
<div class="section level1">
<h1 id="example-simulated-inputs">Example simulated inputs<a class="anchor" aria-label="anchor" href="#example-simulated-inputs"></a>
</h1>
<p>TIN provides a function to generate a random dataset to test the tool. The input <code>TIN</code> and <code>TIT</code> parameters can be specified.</p>
<blockquote>
<p>Note: the sampler generates VAFs using <a href="https://caravagnalab.github.io/mobster/" class="external-link">MOBSTER</a>, and then samples some clonal mutations and contaminates the normal. This process is noisy and might return slightly lower values for the actual <code>TIT</code> estimate. The best setup to test TINC is with a simulated BAM file - e.g., created via <a href="https://github.com/adamewing/bamsurgeon" class="external-link">BAMSurgeon</a>.</p>
</blockquote>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/Random.html" class="external-link">set.seed</a></span><span class="op">(</span><span class="fl">1234</span><span class="op">)</span>

<span class="co"># Sample data from a tumour with 80% purity, and 10% contamination in the normal</span>
<span class="va">sampled_data</span> <span class="op">=</span> <span class="fu"><a href="../reference/random_TIN.html">random_TIN</a></span><span class="op">(</span>TIT <span class="op">=</span> <span class="fl">.8</span>, TIN <span class="op">=</span> <span class="fl">0.1</span><span class="op">)</span>

<span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">sampled_data</span><span class="op">)</span>
<span class="co">#&gt; $data</span>
<span class="co">#&gt; <span style="color: #949494;"># A tibble: 989 × 14</span></span>
<span class="co">#&gt;    chr        from       to ref   alt   simulated_clust… n_tot_count t_tot_count</span>
<span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;int&gt;</span>    <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                  <span style="color: #949494; font-style: italic;">&lt;int&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;int&gt;</span></span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> chr20  31<span style="text-decoration: underline;">766</span>660   3.18<span style="color: #949494;">e</span>7 G     G     C1                        18         119</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> chr12  93<span style="text-decoration: underline;">219</span>100   9.32<span style="color: #949494;">e</span>7 G     G     C1                        27         123</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> chr14  81<span style="text-decoration: underline;">070</span>741   8.11<span style="color: #949494;">e</span>7 T     G     C1                        30         118</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> chr21  18<span style="text-decoration: underline;">665</span>812   1.87<span style="color: #949494;">e</span>7 T     G     C1                        31         121</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> chr11 131<span style="text-decoration: underline;">443</span>607   1.31<span style="color: #949494;">e</span>8 G     T     C1                        26         113</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> chr6   79<span style="text-decoration: underline;">993</span>673   8.00<span style="color: #949494;">e</span>7 T     A     C1                        32         134</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> chr2  186<span style="text-decoration: underline;">544</span>607   1.87<span style="color: #949494;">e</span>8 C     T     C1                        34         112</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> chr7   34<span style="text-decoration: underline;">566</span>354   3.46<span style="color: #949494;">e</span>7 G     T     C1                        29         123</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> chr14  34<span style="text-decoration: underline;">944</span>604   3.49<span style="color: #949494;">e</span>7 A     A     C1                        31         129</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">10</span> chr18  18<span style="text-decoration: underline;">882</span>341   1.89<span style="color: #949494;">e</span>7 C     A     C1                        19         110</span>
<span class="co">#&gt; <span style="color: #949494;"># … with 979 more rows, and 6 more variables: n_alt_count &lt;dbl&gt;,</span></span>
<span class="co">#&gt; <span style="color: #949494;">#   t_alt_count &lt;dbl&gt;, n_ref_count &lt;dbl&gt;, t_ref_count &lt;dbl&gt;, sim_t_vaf &lt;dbl&gt;,</span></span>
<span class="co">#&gt; <span style="color: #949494;">#   sim_n_vaf &lt;dbl&gt;</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $cna</span>
<span class="co">#&gt; <span style="color: #949494;"># A tibble: 989 × 6</span></span>
<span class="co">#&gt;    chr        from        to ref   Major minor</span>
<span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> chr20  31<span style="text-decoration: underline;">766</span>659  31<span style="text-decoration: underline;">766</span>662 G         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> chr12  93<span style="text-decoration: underline;">219</span>099  93<span style="text-decoration: underline;">219</span>102 G         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> chr14  81<span style="text-decoration: underline;">070</span>740  81<span style="text-decoration: underline;">070</span>743 T         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> chr21  18<span style="text-decoration: underline;">665</span>811  18<span style="text-decoration: underline;">665</span>814 T         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> chr11 131<span style="text-decoration: underline;">443</span>606 131<span style="text-decoration: underline;">443</span>609 G         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> chr6   79<span style="text-decoration: underline;">993</span>672  79<span style="text-decoration: underline;">993</span>675 T         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> chr2  186<span style="text-decoration: underline;">544</span>606 186<span style="text-decoration: underline;">544</span>609 C         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> chr7   34<span style="text-decoration: underline;">566</span>353  34<span style="text-decoration: underline;">566</span>356 G         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> chr14  34<span style="text-decoration: underline;">944</span>603  34<span style="text-decoration: underline;">944</span>606 A         1     1</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">10</span> chr18  18<span style="text-decoration: underline;">882</span>340  18<span style="text-decoration: underline;">882</span>343 C         1     1</span>
<span class="co">#&gt; <span style="color: #949494;"># … with 979 more rows</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $plot</span></code></pre></div>
<p><img src="Analysis_files/figure-html/unnamed-chunk-2-1.png" width="576"></p>
<p>The sampler generates random SNVs, and plain diploid CNA segments. These segments are not real and span just through each one of the simulated SNVs.</p>
<p>Data mappings - done internally by TINC - are also available via the <code>load_TINC_input</code> function.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/load_TINC_input.html">load_TINC_input</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">sampled_data</span><span class="op">$</span><span class="va">data</span>, cna <span class="op">=</span> <span class="va">sampled_data</span><span class="op">$</span><span class="va">cna</span><span class="op">)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Loading TINC input data</span> <span style="color: #00BBBB;">─────────────────────────────────────────────────────</span></span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Input data contains n = 989 mutations, selecting operation mode.</span>
<span class="co">#&gt; <span style="color: #BBBB00;">!</span> Found CNA data, retaining only mutations that map to segments with predominant karyotype ...</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #BBBBBB; background-color: #0000BB;"> [ CNAqc - CNA Quality Check ] </span></span>
<span class="co"><span style="color: #BBBBBB; background-color: #0000BB;">#&gt; </span></span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Using reference genome coordinates for: <span style="color: #00BB00;">GRCh38</span>.</span>
<span class="co">#&gt; <span style="color: #BBBB00;">!</span> Missing CCF column from CNA calls, adding CCF = 1 assuming clonal CNA calls.</span>
<span class="co">#&gt; <span style="color: #BBBB00;">!</span> Missing segments length from CNA calls, adding it to CNA calls.</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Input n = 989 mutations for 989 CNA segments (989 clonal, 0 subclonal)</span>
<span class="co">#&gt; Warning in map_mutations_to_segments(snvs, cna %&gt;% filter(CCF == 1)): [CNAqc] a</span>
<span class="co">#&gt; karyotype column is present in CNA calls, and will be overwritten</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Mapped n = 989 mutations to clonal segments (100% of input)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; ── <span style="font-weight: bold;">Genome coverage by karyotype, in basepairs.</span> ──</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #949494;"># A tibble: 1 × 4</span></span>
<span class="co">#&gt;   minor Major     n karyotype</span>
<span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>    </span>
<span class="co">#&gt; <span style="color: #BCBCBC;">1</span>     1     1  <span style="text-decoration: underline;">2</span>967 1:1</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> n = 989 mutations mapped to CNA segments with karyotype <span style="color: #00BB00;">1:1</span> (largest available in basepairs).</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Mutation with VAF within 0 and 0.7 ~ n = 989.</span>
<span class="co">#&gt; $mutations</span>
<span class="co">#&gt; <span style="color: #949494;"># A tibble: 989 × 12</span></span>
<span class="co">#&gt;    chr        from        to ref   alt   n_ref_count n_alt_count t_ref_count</span>
<span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;int&gt;</span>     <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> chr20  31<span style="text-decoration: underline;">766</span>660  31<span style="text-decoration: underline;">766</span>661 G     G              18           0          72</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> chr12  93<span style="text-decoration: underline;">219</span>100  93<span style="text-decoration: underline;">219</span>101 G     G              26           1          72</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> chr14  81<span style="text-decoration: underline;">070</span>741  81<span style="text-decoration: underline;">070</span>742 T     G              29           1          65</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> chr21  18<span style="text-decoration: underline;">665</span>812  18<span style="text-decoration: underline;">665</span>813 T     G              30           1          72</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> chr11 131<span style="text-decoration: underline;">443</span>607 131<span style="text-decoration: underline;">443</span>608 G     T              25           1          65</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> chr6   79<span style="text-decoration: underline;">993</span>673  79<span style="text-decoration: underline;">993</span>674 T     A              31           1          82</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> chr2  186<span style="text-decoration: underline;">544</span>607 186<span style="text-decoration: underline;">544</span>608 C     T              33           1          62</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> chr7   34<span style="text-decoration: underline;">566</span>354  34<span style="text-decoration: underline;">566</span>355 G     T              28           1          77</span>
<span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> chr14  34<span style="text-decoration: underline;">944</span>604  34<span style="text-decoration: underline;">944</span>605 A     A              30           1          83</span>
<span class="co">#&gt; <span style="color: #BCBCBC;">10</span> chr18  18<span style="text-decoration: underline;">882</span>341  18<span style="text-decoration: underline;">882</span>342 C     A              19           0          77</span>
<span class="co">#&gt; <span style="color: #949494;"># … with 979 more rows, and 4 more variables: t_alt_count &lt;dbl&gt;,</span></span>
<span class="co">#&gt; <span style="color: #949494;">#   karyotype &lt;chr&gt;, id &lt;chr&gt;, OK_tumour &lt;lgl&gt;</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $cna_map</span>
<span class="co">#&gt; ── <span style="color: #000000; background-color: #BBBB00;">[ CNAqc ] </span> n = 989 mutations in <span style="color: #00BB00;">989</span> segments (989 clonal + 0 subclonal). Geno</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;  1:1  [n = 989, L =   0 Mb] ■■■■■■■■■■■■■■■■■■■■■■■■■■■</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Sample Purity: 80% ~ Ploidy: 2.</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; $what_we_used</span>
<span class="co">#&gt; [1] "1:1"</span></code></pre></div>
<p>TINC fits the data via the <code>autofit</code> function.</p>
<p>A flag <code>FAST</code> sets faster set of parameters to determine TIN and TIT scores - we usually this for prototyping analyses and then turn it to <code>FALSE</code>.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># Autofit function</span>
<span class="va">TINC_fit</span> <span class="op">=</span> <span class="fu"><a href="../reference/autofit.html">autofit</a></span><span class="op">(</span><span class="va">sampled_data</span><span class="op">$</span><span class="va">data</span>, cna <span class="op">=</span> <span class="cn">NULL</span>, FAST <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span>
<span class="co">#&gt; <span style="color: #BBBBBB; background-color: #0000BB;"> [ TINC ] </span></span>
<span class="co"><span style="color: #BBBBBB; background-color: #0000BB;">#&gt; </span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Loading TINC input data</span> <span style="color: #00BBBB;">─────────────────────────────────────────────────────</span></span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Input data contains n = 989 mutations, selecting operation mode.</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Mutation with VAF within 0 and 0.7 ~ n = 989.</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Analysing tumour sample with MOBSTER</span> <span style="color: #00BBBB;">────────────────────────────────────────</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #BBBBBB; background-color: #0000BB;"> [ MOBSTER fit ] </span></span>
<span class="co"><span style="color: #BBBBBB; background-color: #0000BB;">#&gt; </span></span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Loaded input data, n = 989.</span>
<span class="co">#&gt; ❯ n = 989. Mixture with k = <span style="color: #00BB00;">1,2</span> Beta(s). Pareto tail: <span style="color: #00BB00;">TRUE</span> and <span style="color: #00BB00;">FALSE</span>. Output</span>
<span class="co">#&gt; clusters with π &gt; 0.02 and n &gt; 10.</span>
<span class="co">#&gt; <span style="color: #BBBB00;">!</span> mobster automatic setup <span style="color: #00BB00;">FAST</span> for the analysis.</span>
<span class="co">#&gt; ❯ Scoring (<span style="color: #BB0000;">without parallel</span>) 2 x 2 x 2 = <span style="color: #00BB00;">8</span> models by <span style="color: #00BB00;">reICL</span>.</span>
<span class="co">#&gt; <span style="color: #BBBBBB;">ℹ</span> <span style="font-weight: bold;">MOBSTER fits</span> completed in 13.9s.</span>
<span class="co">#&gt; ── <span style="color: #000000; background-color: #BBBB00;">[ MOBSTER ] My MOBSTER model</span> n = 989 with <span style="color: #00BB00;">k = 1</span> Beta(s) <span style="color: #00BB00;">and a tail</span> ──────────</span>
<span class="co">#&gt; ● Clusters: π = 70% [<span style="color: #BBBB00;">C1</span>] and 30% [<span style="color: #BBBB00;">Tail</span>], with π &gt; 0.</span>
<span class="co">#&gt; ● Tail [n = 281, 30%] with alpha = 1.2.</span>
<span class="co">#&gt; ● Beta <span style="color: #00BB00;">C1</span> [n = 708, 70%] with mean = 0.4.</span>
<span class="co">#&gt; <span style="color: #BBBBBB;">ℹ</span> Score(s): NLL = -1066.18; ICL = -2014.36 (-2090.98), H = 76.61 (0). Fit</span>
<span class="co">#&gt; <span style="color: #00BB00;">converged</span> by <span style="color: #00BB00;">MM</span> in 13 steps.</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Using the location likelihood heuristic to inspect mutations' distribution</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Cluster C1: 60% of counts in 12 chromosomes (spread over &gt;20% of used chromosomes: 4.4).</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Without CNA, TINC will estimate tumour purity as 2*x, with x the clonal peak.</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> MOBSTER found n = 702 clonal mutations from cluster C1</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Analysing normal sample with BMix</span> <span style="color: #00BBBB;">───────────────────────────────────────────</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">BMix fit</span> <span style="color: #00BBBB;">────────────────────────────────────────────────────────────────────</span></span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Binomials k_B = <span style="color: #00BB00;">1</span> and <span style="color: #00BB00;">2</span>, Beta-Binomials k_BB = <span style="color: #00BB00;">0</span>; 4 fits to run.</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> <span style="font-weight: bold;">Bmix best fit</span> completed in 0.01 mins</span>
<span class="co">#&gt; ── <span style="color: #000000; background-color: #BBBB00;">[ BMix ] My BMix model</span> n = 702 with <span style="color: #00BB00;">k = 1</span> component(s) (<span style="color: #00BB00;">1</span> + <span style="color: #00BB00;">0</span>) ──────────────</span>
<span class="co">#&gt; • Clusters: π = 100% [<span style="color: #BBBB00;">Bin 1</span>], with π &gt; 0.</span>
<span class="co">#&gt; • Binomial <span style="color: #00BB00;">Bin 1</span> with mean = 0.034459749296983.</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Score (model selection): <span style="color: #00BB00;">ICL</span> = 1512.24.</span>
<span class="co">#&gt; Scale for 'x' is already present. Adding another scale for 'x', which will</span>
<span class="co">#&gt; replace the existing scale.</span>
<span class="co">#&gt; Scale for 'fill' is already present. Adding another scale for 'fill', which</span>
<span class="co">#&gt; will replace the existing scale.</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Binomial peaks 0.034459749296983 with proportions 1. Clonal score 0.034459749296983 with TINN 0.068919498593966</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">Analysing tumour and normal samples with VIBER</span> <span style="color: #00BBBB;">──────────────────────────────</span></span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #BBBBBB; background-color: #0000BB;"> [ VIBER - variational fit ] </span></span>
<span class="co"><span style="color: #BBBBBB; background-color: #0000BB;">#&gt; </span></span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Input n = 989, with k &lt; 5. Dirichlet concentration <span style="color: #00BB00;">α = 1e-06</span>.</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Beta (a_0, b_0) = (1, 1); q_i = prior. Optimise: ε = 1e-06 or 1000 steps, r = 3 starts.</span>
<span class="co">#&gt; [easypar] 2022-01-12 10:21:46 - Overriding parallel execution setup [TRUE] with global option : FALSE</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> <span style="font-weight: bold;">VIBER fit</span> completed in 0.04 mins (status: <span style="color: #00BB00;">converged</span>)</span>
<span class="co">#&gt; ── <span style="color: #000000; background-color: #BBBB00;">[ VIBER ] My VIBER model</span> n = 989 (w = 2 dimensions). Fit with <span style="color: #00BB00;">k = 5</span> clusters.</span>
<span class="co">#&gt; • Clusters: π = 73% [<span style="color: #BBBB00;">C1</span>], 25% [<span style="color: #BBBB00;">C4</span>], and 2% [<span style="color: #BBBB00;">C3</span>], with π &gt; 0.</span>
<span class="co">#&gt; • Binomials: θ = &lt;0.03, 0.4&gt; [<span style="color: #BBBB00;">C1</span>], &lt;0, 0.09&gt; [<span style="color: #BBBB00;">C4</span>], and &lt;0, 0.21&gt; [<span style="color: #BBBB00;">C3</span>].</span>
<span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Score(s): ELBO = -140395.831. Fit <span style="color: #00BB00;">converged</span> in 47 steps, ε = 1e-06.</span>
<span class="co">#&gt; <span style="color: #00BB00;">✔</span> Reduced to k = 3 (from 5) selecting VIBER cluster(s) with π &gt; 0.02, and Binomial p &gt; 0 in w &gt; 0 dimension(s).</span></code></pre></div>
<p>Summary of the results (with an S3 object)</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">TINC_fit</span><span class="op">)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #00BBBB;">──</span> <span style="font-weight: bold;">TINC profiler for bulk WGS</span> <span style="color: #00BBBB;">──────────────────────────────────────────────────</span></span>
<span class="co">#&gt; <span style="color: #BBBB00;">!</span> Copy Number data has not been used for this analysis.</span>
<span class="co">#&gt; → Mutations data: n = <span style="color: #00BB00;">989</span> out of 989 within range (<span style="color: #00BB00;">100%</span>).</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; <span style="color: #000000; background-color: #BBBB00;">        TIT : </span> 81% (RF 81)  ~ n = 702 clonal mutations, cluster C1 </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt;         TIN : </span> 7% (RF 7)  ~ n = 671 with VAF &gt; 0 </span>
<span class="co"><span style="color: #000000; background-color: #BBBB00;">#&gt; </span></span>
<span class="co">#&gt; <span style="color: #000000; background-color: #0000BB; font-weight: bold;">   QC Tumour  </span> <span style="color: #0000BB;">Good purity (65-85%)</span></span>
<span class="co">#&gt; <span style="color: #000000; background-color: #BBBB00; font-weight: bold;">   QC Normal  </span> <span style="color: #BBBB00;">Some contamination (3-7%)</span></span></code></pre></div>
<p>Plot the results (with an S3 object).</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">TINC_fit</span><span class="op">)</span></code></pre></div>
<p><img src="Analysis_files/figure-html/unnamed-chunk-6-1.png" width="960"></p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R">
<span class="co"># Analogous to</span>
<span class="co"># plot_simple_report(TINC_fit)</span></code></pre></div>
<p>Or make a more detailed report</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/plot_full_page_report.html">plot_full_page_report</a></span><span class="op">(</span><span class="va">TINC_fit</span><span class="op">)</span></code></pre></div>
<p><img src="Analysis_files/figure-html/unnamed-chunk-7-1.png" width="960"></p>
<p>A similar fit object is available inside the package (called <code>fit_example</code>).</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by <a href="https://www.caravagnalab.org" class="external-link">Giulio Caravagna</a>.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.1.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
